src/CrystallineSociety/Client/Web/Pages/_Layout.cshtml

```diff
@@ -49,11 +49,18 @@
         <link rel="manifest" href="manifest.json">
     }
     <style>
-        html, body, #app-container {
+        html {
+            margin: 0;
+            padding: 0;
+            font-size: 62.5%;
+        }
+
+        body, #app-container {
             margin: 0;
             padding: 0;
             width: 100%;
-            height: 100%;
```

Comment:
We must use container of Bootstrap

Reply:
This is in _Layout.cshtml. It's better to be like this.